traveler_ui_kit/src/main/java/com/guestlogix/traveleruikit/fragments/ParkingActionStripFragment.java

```diff
@@ -0,0 +1,110 @@
+package com.guestlogix.traveleruikit.fragments;
+
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import com.guestlogix.travelercorekit.callbacks.FetchPurchaseFormCallback;
+import com.guestlogix.travelercorekit.models.Price;
+import com.guestlogix.travelercorekit.models.PurchaseForm;
+import com.guestlogix.travelercorekit.models.Traveler;
+import com.guestlogix.travelercorekit.utilities.Assertion;
+import com.guestlogix.traveleruikit.R;
+import com.guestlogix.traveleruikit.TravelerUI;
+import com.guestlogix.traveleruikit.activities.AvailabilityActivity;
+import com.guestlogix.traveleruikit.activities.QuestionsActivity;
+import com.guestlogix.traveleruikit.models.BookingContext;
+import com.guestlogix.traveleruikit.widgets.ActionStrip;
+
+import java.util.ArrayList;
+import java.util.Locale;
+
+import static com.guestlogix.traveleruikit.activities.OrderConfirmationActivity.REQUEST_CODE_ORDER_FLOW;
+import static com.guestlogix.traveleruikit.activities.OrderConfirmationActivity.RESULT_OK_ORDER_CONFIRMED;
+
+/**
+ * Fragment to handle Bookable item actions
+ */
+public class ParkingActionStripFragment extends Fragment implements FetchPurchaseFormCallback {
```

Comment:
Why couldnt you use the existing action strip? The action strip should eventually be gotten rid of, but in the mean time there is no reason why you cant use the same strip for multiple purposes...it is agnostic of what it is used for

Reply:
Lets not create new ones

Reply:
I will merge Parking, Bookable and Buyable ActionStripFragments into one and reconsider the params.

Reply:
✓